### PR TITLE
Enhances #1641 : Embedded ASCII Cast of Complete Process

### LINKED ISSUE #1663: Add Docker Installation Procedure
**Actual Behaviour**

No such documentation exists, anywhere !

**Expected Behaviour**

It should !

**Would you like to work on the issue?**

**Just firing the PR!**

### PR DESCRIPTION
**Pull request by 0x48piraj**
# Enhances #1641

Changes: 

- Embedded ASCII Cast of Complete Process

Screenshots for the change: 

image

### CODE REVIEW

**Comment by 0x48piraj** [comment]
Ping @geekyd

File: docs/INSTALLATION_GOOGLE.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # How to install the Open Event Web App Generator on Google Cloud
 This documentation provides instructions to deploy the Open Event Web App on Google Cloud using the App Engine flexible environment.
 
+[![ASCII Cast of Complete Process](https://asciinema.org/a/156500.png)](https://asciinema.org/a/156500)
```
Comment (geekyd) [inline docs/INSTALLATION_GOOGLE.md]: I do not think adding an image instead of link is helpful. Please use the link only

Comment (0x48piraj) [reply docs/INSTALLATION_GOOGLE.md]: You sure ?
In other setup docs, excessive uses of images are used i.e. heroku

I'm just asking, will do after your reply. :blush:

Comment (geekyd) [reply docs/INSTALLATION_GOOGLE.md]: @0x48piraj Yes, we do not need to add a screenshot of the video to link it. Please use the link only. If possible please improve the quality of the docs.

**Pull request closed without merging**